load_map_data.sh

```diff
@@ -63,17 +63,16 @@ if [ ! -d "${shape_path}" ]; then
   unzip "*.zip"
   find -iname '*.shp' -execdir shapeindex {} \;
 
-  cp /style/themes/$STYLE_DIR/*.mss /style/themes/osm-smartrak
+  cp -r /style/themes/$STYLE_DIR /style/themes/target
```

Comment:
thought (non-blocking):

> I opted to not do the parts about using STYLE_DIR

I agree with that choice! 
thought (non-blocking) To me, it would be ideal to not have to copy the targeted style into a separate directory here for the purposes of `make.py` being able to reference a single pre-configured value for `config["theme"]`.
 
 It could be worthwhile to keep this repo set up for supporting meaningfully named styles (rather than "skate" and the "default" which as you point out isn't so descriptive) rather than sprinkling in the new also not-so-descriptive "TargetTheme". Maybe we could adopt some sort of major map version naming convention that is agnostic to where it will be used. But that is tangential to this PR, and maybe for the consolidation discussion!

Reply:
Yeah, I think we'll have a better sense of next steps here once we talk to @thecristen about it this week.